docs/guide/multisite/site_factory.md

```diff
@@ -0,0 +1,216 @@
+    system:
+        example_site_factory_group_1:
+            design: example_1
+        example_site_factory_group_2:
+            design: example_2
+```
+
+### Add site template configuration
+
+Add thumbnails and names for your site templates in `config/packages/ezplatform_site_factory.yaml`.
+
+```yaml
+ez_platform_site_factory:
+    templates:
+        site1:
+            siteaccess_group: example_site_factory_group_1
+            name: example_site_1
+            thumbnail: /path/to/image/example-thumbnail_1.png
+        site2:
+            siteaccess_group: example_site_factory_group_2
+            name: example_site_2
+            thumbnail: /path/to/image/example-thumbnail_2.png
+```
+
+You can check the results of your work in the Back Office by going to the **Site** tab and selecting the **List** icon.
+
+There, you should be able to add a new site and choose a design for it.
+
+### Define domains
+
+To be able to see your site online, you need to define a domain for it.
+
+!!! caution "Define domain for production environment"
+
+    These steps are for `dev` environment only.
+    If you want to define domains in production environment, you will need to configure Apache or Nginx by yourself.
+
+In the `.env` file change line 2 to: `COMPOSE_FILE=doc/docker/base-dev.yml:doc/docker/multihost.yml`
+
+Take a look into the `doc/docker/multihost.yml` file. Here you can define domains.
+To add a new domain, add it in `command:` and under frontend and backend aliases as shown in the example below:
+
+```yaml hl_lines="3 6 11"
+services:
+  web:
+    command: /bin/bash -c "cd /var/www && cp -a doc/nginx/ez_params.d /etc/nginx && bin/vhost.sh --host-name=site.example.com --host-alias='admin.example.com test.example.com' --template-file=doc/nginx/vhost.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+    networks:
+      frontend:
+        aliases:
+          - site.example.com
+          - admin.example.com
+          - test.example.com
+      backend:
+        aliases:
+          - site.example.com
+          - admin.example.com
+          - test.example.com
+```
+
+Next, you must define the domains in `etc/hosts`:
+
+`0.0.0.0 site.example.com admin.example.com test.example.com www.admin.example.com`
+
+Then, run `docker-compose up`:
+
+```bash
+export COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/multihost.yml"
+docker-compose up
+```       
+
+Your sites should be now visible under:
+
+- `http://site.example.com:8080/`
+- `http://admin.example.com:8080/`
+- `http://localhost:8080/`
+- `http://test.example.com:8080/`
+
+![Site Factory enabled](../img/site_factory_site_list.png "Site Factory enabled")
+
+### Define site directory
+
+You can adjust the place where the directory of the new site is created (Location with ID 2 by default).
+To do it, go to `config/packages/ezplatform_site_factory.yaml`, and add the following parameter:
+
+``` yaml
+parameters:
+    ezsettings.default.site_factory.sites_location_id: 42
```

Comment:
@mikadamczyk is this really a parameter or is there real semantic config for that?

Reply:
Hi @alongosz! It is a real semantic configuration https://github.com/ezsystems/ezplatform-site-factory/blob/master/src/bundle/DependencyInjection/Parser/SiteFactory.php#L35

Reply:
@mikadamczyk Shouldn't I then replace it with 'regular' config?
```
ezplatform:
    system:
        default:
            site_factory:
                sites_location_id: 2
```

Reply:
It definitely cannot be documented as a parameter in this case.

Reply:
Yep, already replaced with the example above.